index.bs

```diff
@@ -1263,6 +1263,7 @@ The [=authenticator data=] structure is a byte array of 37 bytes or more, as fol
         <td>
             Flags (bit 0 is the least significant bit):
             - Bit 0: [=Test of User Presence=] (`TUP`) result.
+            - Bit 1: [=User Verification=] (`UV`) result. 
             - Bits 1-5: Reserved for future use (`RFU`).
```

Comment:
Note: this needs to be updated [per @jyasskin's earlier comment] to be "Bits 2-5" !